react-native-libraries.json

```
@@ -13123,6 +13123,38 @@
     "ios": true,
     "android": true,
     "expoGo": true,
+  },
+    "githubUrl": "https://github.com/mrousavy/nitro/tree/main/packages/react-native-nitro-modules",
```

```suggestion
  },
  {
    "githubUrl": "https://github.com/mrousavy/nitro/tree/main/packages/react-native-nitro-modules",
```

The opening bracket is missing, which throws syntax error in CI.

---

sorry my bad did the merge on my phone